package-resources.yml

```diff
@@ -0,0 +1,53 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  creationTimestamp: null
+  name: kapp-controller.carvel.dev.0.0.0
+spec:
+  refName: kapp-controller.carvel.dev
+  releasedAt: null
+  template:
```

Comment:
I think we can add License to the package as well. Something similar we have in carvel [secretgen-controller](https://github.com/vmware-tanzu/carvel-secretgen-controller/blob/91aaaeb82515be5e6dd07689dbd55c0d1c9b4f2f/package-resources.yml#L8)

Reply:
Done.